core/state_transition.go

```diff
@@ -649,6 +666,16 @@ func (st *StateTransition) refundGas(refundQuotient uint64) uint64 {
 		st.evm.Config.Tracer.OnGasChange(st.gasRemaining, 0, tracing.GasChangeTxLeftOverReturned)
 	}
 
```

Comment:
This is the code path we would disable if there was an issue with denial of service